examples/heif_thumbnailer.cc

```diff
@@ -42,7 +42,7 @@
 #  include "encoder_png.h"
 #endif
 
-#if defined(_MSC_VER)
+#if defined(__MINGW32__) || defined(__MINGW64__) || defined(_MSC_VER)
```

Comment:
I think you can just `#ifdef _WIN32`

Reply:
Thanks, I was thinking about that too, but then wasn't be sure one wants this for Cygwin/MSYS as well (perhaps multibyte UTF8 just works there)?

In either case, still stuck w/ the older libstdc++ problem...

W/ the separate MSVC/MINGW, one could potentially also check for `__GLIBCXX__` version...

Reply:
I don't think that's relevant on cygwin, locale encoded filenames should just work there.